src/dmd/backend/cod4.d

```diff
@@ -876,40 +885,48 @@ else
     uint jop;
 
 
-  switch (op)                   // select instruction opcodes
-  {     case OPpostinc: op = OPaddass;                  // i++ => +=
+    switch (op)                   // select instruction opcodes
+    {
+        case OPpostinc: op = OPaddass;                  // i++ => +=
```

Comment:
I would move down the code to below the case statement. Now the code gets quite right adjusted.